src/agent/src/agent_registration.cpp

```diff
@@ -1,8 +1,8 @@
 #include <agent_registration.hpp>
 
-#include <logger.hpp>
-
 #include <boost/beast/http.hpp>
+#include <fmt/format.h>
```

Comment:
It looks like if we are going to use fmt it should be included privately in the target include libraries of the Agent target. From what I gather, fmt is a public dependency of Logger... but it is actually not used at all by Logger... I may be wrong, but we could try removing fmt::format from Logger and if we actually need it here, add it privately to the Agent target.

Reply:
Yes, I was able to remove `fmt` from the `Logger` and include it as a private library in the `Agent` without problems! I also stopped using fmt for these simpler messages.